rfc/rfc-8/rfc-8.md

```diff
@@ -0,0 +1,209 @@
+<!--
+  Licensed to the Apache Software Foundation (ASF) under one or more
+  contributor license agreements.  See the NOTICE file distributed with
+  this work for additional information regarding copyright ownership.
+  The ASF licenses this file to You under the Apache License, Version 2.0
+  (the "License"); you may not use this file except in compliance with
+  the License.  You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+-->
+# RFC-8: Metadata based Record Index
+
+## Proposers
+- @prashantwason
+
+## Approvers
+
+
+## Status
+JIRA: https://issues.apache.org/jira/browse/HUDI-53
+
+
+## Abstract
+HUDI requires an [Index](https://hudi.apache.org/docs/indexing) during updates to locate the existing records by their 
+unique record keys. The HUDI Index saves a mapping of the record-key to record's file path. Hudi supports several indexes 
```

Comment:
nit: The HUDI Index saves a mapping of the record-key to record's file path. ->
The HUDI Index serves a mapping of the record-key to record's file path.